### DISCUSSION TITLE
Can I use postgresql "CURRENT_TIMESTAMP" as a default value?

### DISCUSSION THREAD

**Discussion by GiyoMoon** [SeaQL/sea-query#346] [Q&A] [answered]
Hey, I'm using sea-orm with sea-query to create migrations which set up my postgresql database. I was wondering if it is possible to set `CURRENT_TIMESTAMP` as the default value for a timestamp colum. If yes, how would I do this?

```rust
Table::create()
    .table(User::Entity)
    .if_not_exists()
    .col(ColumnDef::new(User::Id).uuid().not_null().primary_key())
    .col(
        ColumnDef::new(User::CreatedAt)
            .timestamp()
            .default()  // <== HERE, I would like to set CURRENT_TIMESTAMP as the default value
            .not_null(),
    );

#[derive(Iden)]
pub enum User {
    Entity,
    Id,
    CreatedAt,
}
```
Should generate:
```sql
CREATE TABLE user (
    id UUID PRIMARY KEY
    created_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP
);
```
Btw awesome query builder and ORM crates! I love using them :)

**Comment by billy1624** [comment, marked as answer]
Hey @GiyoMoon, currently there is no official way of doing it. I just opened an issue tracking this:
- https://github.com/SeaQL/sea-query/issues/347

At the meantime, you can use `extra` method as a workaround:
- https://github.com/SeaQL/sea-query/blob/6c9527ae08e32c55490ae510be41f646e030c2a2/tests/mysql/table.rs#L114-L133
- e.g. `.extra("DEFAULT CURRENT_TIMESTAMP".to_string())`

**Reply by GiyoMoon** [reply]
Exactly what I need. Thank you very much!

**Reply by billy1624** [reply]
Welcome! Stay tuned for https://github.com/SeaQL/sea-query/issues/347
